modules/setting/setting.go

```diff
@@ -446,11 +446,12 @@ func newSessionService() {
 
 // Mailer represents mail service.
 type Mailer struct {
-	Name         string
-	Host         string
-	From         string
-	User, Passwd string
-	SkipVerify   bool
+	Name              string
+	Host              string
+	From              string
+	User, Passwd      string
+	SkipVerify        bool
+	CertFile, KeyFile string
```

Comment:
Corresponding to above comment, here we may also want a `UseCertificates` field.